src/System.Reflection.Metadata/src/System/Reflection/Metadata/IL/MethodBodyBlock.cs

```diff
@@ -153,7 +153,7 @@ public static MethodBodyBlock Create(BlobReader reader)
             }
 
             var ilBlock = reader.GetMemoryBlockAt(0, ilSize);
-            reader.SkipBytes(ilSize);
+            reader.Offset += ilSize;
```

Comment:
Does set_Offset handle checking for overflows, or maybe that's not necessary here?